src/store/annotations/reducer.ts

```diff
@@ -15,6 +19,7 @@ const annotationsById = createReducer<AnnotationsState['byId']>({}, builder =>
 );
 
 export default combineReducers({
+    activeAnnotationId,
```

Comment:
We shouldn't need to prefix things within the reducer itself (`byId` vs `annotationsById`). Can this be `activeId`?